x/ibc/handler.go

```diff
@@ -0,0 +1,13 @@
+package ibc
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// NewHandler defines the IBC handler
+func NewHandler(k Keeper) sdk.Handler {
```

Comment:
Should this handler route to the submodules? Or maybe all of the submodule handlers are added here? What should this file do?

Reply:
updated